_topic_maps/_topic_map.yml

```
@@ -1227,6 +1227,12 @@ Topics:
     File: cert-manager-log-levels
   - Name: Uninstalling the cert-manager Operator for Red Hat OpenShift
     File: cert-manager-operator-uninstall
+- Name: External secrets Operator for Red Hat OpenShift
```

@aravipra From a strictly grammatical perspective, you can use lowercase here, as the first word is supposed to always be lowercase. I do not see the name with _External_ capitalized in the [Do Not Use](https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit?gid=1375785039#gid=1375785039&range=F113) list in the Official Red Hat product names list. Maybe you can ask your team about this? 
```suggestion
- Name: external secrets Operator for Red Hat OpenShift
```